# AUTHOR: Neset Unver Akmandor (NUA)
# E-MAIL: akmandor.n@northeastern.edu

ws_name: "mobiman_simulation"

flag_save: false
flag_publish_frame: true

trajectory_name: "jackal"

#trajectory_data_path: ""
trajectory_data_path: "dataset/trajectory_sampling/jackal/20240312_153905/"
flag_read_only: true

trajectory_sampling_dataset_path: "dataset/trajectory_sampling/jackal/"

tss_robot_frame_name: "base_link"

# geometric
# kinematic
trajectory_gen_type: "kinematic"

## Geometric Parameters ~~~~ 
# cone
# cube
#geo_type: "cube"

## Cone Parameters ~~~~ 
#tlen: 0.0
#tyaw: 0.9       # rad
#tpitch: 0.6     # rad
#tyaw_cnt: 21
#tpitch_cnt: 11
#tsamp_cnt: 3

## Cube Parameters ~~~~ 
#x_min: 0.0
#x_max: 0.0
#x_cnt: 0.0

#y_min: 0.0
#y_max: 0.0
#y_cnt: 0.0

#z_min: 0.0
#z_max: 0.0
#z_cnt: 0.0
## ~~~~~~~~~~~~~~~~~~~~~~~~~

## Kinematic Parameters ~~~~ 
tsamp_cnt: 5
dt: 0.01
ttime: 1.5
lat_velo_samp_cnt: 2
ang_velo_samp_cnt: 5
min_lat_velo: -1.0
max_lat_velo: 1.0
max_yaw_velo: 2.0
sample_start_index: 2
## ~~~~~~~~~~~~~~~~~~~~~~~~~